pkg/mutators/podtemplatespec.go

```diff
@@ -127,6 +127,8 @@ func (p *PodTemplateSpec) setMemoryRequest(ctx context.Context, container *corev
 	if !calculatedRequest.IsZero() {
 		if p.dryRun {
 			log.Info(fmt.Sprintf("[dry-run] setting memory request to %s", calculatedRequest.String()))
+		} else {
+			log.Info(fmt.Sprintf("setting memory request to %s", calculatedRequest.String()))
```

Comment:
It seems like dry-run logging could be a [field on our logger](https://pkg.go.dev/go.uber.org/zap#Fields) configuration at creation time and we can get rid of all the dryrun checks if we want to always log.

Reply:
Good point, changed to add dry-run as a field in the logger.

```
[hedgetrimmer] {"level":"info","ts":1668446019.9043307,"msg":"setting memory request to 64Mi","dry-run":false,"resource":{"group":"apps","version":"v1","resource":"deployments"},"namespace":"devops","name":"test-deployment","operation":"CREATE"}
[hedgetrimmer] {"level":"info","ts":1668446019.904406,"msg":"setting memory limit to 70Mi","dry-run":false,"resource":{"group":"apps","version":"v1","resource":"deployments"},"namespace":"devops","name":"test-deployment","operation":"CREATE"}
```